ui/selectors/multichain.ts

```diff
@@ -184,6 +185,110 @@ export function getMultichainNetwork(
   };
 }
 
+export function caipChainIdToHex(chainId: CaipChainId): string {
+  const [namespace, reference] = chainId.split(':');
+  if (namespace === 'eip155') {
+    const chainIdDecimal = parseInt(reference, 10);
+    return `0x${chainIdDecimal.toString(16)}`;
+  }
+  return chainId;
+}
+
+export function hexToCaipChainId(hex: string): CaipChainId {
+  const hexValue = hex.startsWith('0x') ? hex.slice(2) : hex;
+  const chainIdDecimal = parseInt(hexValue, 16);
+  return `eip155:${chainIdDecimal}`;
+}
+
+export function getChains(state: MultichainState): MultichainNetwork[] {
```

Comment:
Whats the reason for the changes in this file?  I understood it was for non-evm support, so wondering why it needs to change vs just having a selector in `selectors.js`

Reply:
I moved this to selectors.js

Reply:
I moved this to selectors.js